Joachim development

JoachimIsaac opened the pull request:
I modified the README.md fIle so that it would describe my repo more.

JoachimIsaac committed:
Update the README.md file to describe the repo.

JoachimIsaac committed:
Update the README.md file to describe the repo.

stekolla commented on README.md:
```diff
@@ -1,12 +1,9 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+# This repo contains [Joachim Isaac's](https://ssh.cloud.google.com/cloudshell/editor) portfolio and SPS projects.
```
Maybe have this link to your Github or LinkedIn account instead of the cloud shell editor.

JoachimIsaac replied:
Thank you I missed that, my bad.

stekolla commented on README.md:
```diff
@@ -1,12 +1,9 @@
-Then follow the on-screen instructions to set up your repo.
+Login to GitHub
+Clone this repo: by clicking the clone button
+If your using git through your terminal use: git clone https://github.com/YOUR-USERNAME/YOUR-REPOSITORY
```
Update the URL to point to this repo: https://github.com/JoachimIsaac/software-product-sprint

stekolla replied:
The first "your" should be "you're"

stekolla reviewed (changes requested):
I've added a couple suggestions for the change

JoachimIsaac committed:
Fixed the errors I had in the README file. Then I added links to my relevant sites.

JoachimIsaac commented:
I fixed the errors I had in the README file previously. Then I added links to my relevant sites(Github, LinkedIn, My Youtube Channel and my personal website). I also opted to add a banner at the top of the file.

JoachimIsaac committed:
Need to change the last paragraphs spacing for readability

stekolla reviewed (approved):
Looks good to me
